scripts/Diffraction/isis_powder/routines/common_output.py

```diff
@@ -8,46 +8,52 @@
 import os
 
 
-def split_into_tof_d_spacing_groups(run_details, processed_spectra):
+def split_into_tof_d_spacing_q_squared_groups(run_details, processed_spectra):
```

Comment:
Just a quick question. Are we now doing more work than is necessary for the GEM case where I think we drop the Q^2 output? I may have missed that this is now a requirement but I just wanted to check that we don't want a flag to skip the Q^2 conversion when not required?

Reply:
@DannyHindson would Q^2 be useful for GEM?

Reply:
I'm not in touch with anyone from GEM so I think we should probably leave their process alone. So a flag to skip the Q^2 processing would be good I think